spec/steps/execute.fmf

```diff
@@ -11,29 +11,35 @@ description: |
     in the following format::
 
         /test/one:
-            result: TEST_RESULT
+            result: OUTCOME
             log: PATH
 
         /test/two:
-            result: TEST_RESULT
+            result: OUTCOME
             log: PATH
 
-    Where ``TEST_RESULT`` is the outcome of the test. It can have
-    these values:
+    Where ``OUTCOME`` is the result of the test execution. It can
+    have the following values:
 
-        * info - test finished and produced only information
-          message
-        * passed - test finished and passed
-        * failed - test finished and failed
-        * error - a problem encountered during test execution,
-          treated as failed
+        pass
+            Test execution successfully finished and passed.
+        info
+            Test finished but only produced an informational
+            message. Represents a soft pass, automation should
+            treat this as a passed test and flag for human review.
```

Comment:
Miro, I remember you mentioned some real-life use case for this status. Could you share that example so we get more tangible idea of this will be used?

Reply:
Right, the example we now use is skipped testing. Current examples of usage of this state for the whole run:

1. no baseline build for rpminspect comparison exists (new package or no available tag inheritance to find it) -> not possible to perform test, but it is ok
2. no reverse dependency tests found for the component

Reply:
Great, thanks for clarification!

Reply:
Thinking a bit more, both examples basically mean that the test has been skipped so `skip` would be a better name? Or are there possible other `info` cases when test is performed?

Reply:
Kept `info` as we might use is for `result: ignore` handling as well.